massgov/mayflower-patternlab
# DP-8602: Remove unused slick plugin

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Removes Slick Sllideshow javascript library, which wasn't being used.  

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8602)

## Steps to Test


1. Load the Mayflower start page and ensure that no errors are thrown in the javascript console.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
